packages/core/src/lib/oauth/callback.ts

```diff
@@ -165,6 +166,11 @@ export async function handleOAuth(
     }
   }
 
+  if (tokens.expires_in) {
+    tokens.expires_at =
+      Math.floor(Date.now() / 1000) + Number(tokens.expires_in)
```

Comment:
Since we don't use `openid-client` anymore, [this](https://github.com/panva/node-openid-client/blob/d3642296a08da05c1c2ecce8a3145b2096df20f4/lib/token_set.js#L14-L20) is implemented here now.

Note, we do not save this [by default](https://github.com/nextauthjs/next-auth/pull/7369/files#diff-afe79f6a47742706319241dd1ff8723b7f5aae2723aa4b2c4fca8560d1503d16R110-R116) yet, as refresh token rotation is not currently added, but we provide this value to ease migration/support user-land rotation.